guide/english/computer-science/garbage-collection/index.md

```diff
@@ -5,16 +5,16 @@ title: Garbage Collection
 
 #### What is Garbage Collection?
 
-In general layman term, Garbage collection (GC) is nothing but collecting or gaining the memory back which has been allocated to the objects, which is not in use at the moment in any part of our program. A brief description as below.
+In layman's terms, Garbage collection (GC) is nothing more than collecting or gaining the memory back which has been allocated to objects, which is not in use at the moment in any part of a program. A brief description is below.
 
-Garbage collection is the process in which programs try to free up memory space that is no longer used by objects, and such.  Garbage collection is implemented differently for every language.  Most high-level programming languages have some sort of garbage collection built it.  Low-level programming languages may add garbage collection through libraries.
+Garbage collection is the process in which programs try to free up memory space that is no longer used by objects.  Garbage collection is implemented differently for every language.  Most high-level programming languages have some sort of garbage collection built it.  Low-level programming languages may add garbage collection through libraries.
 
-As said above, every programming language has their own way of GC. In C programming, developers need to take care of memory allocation and deallocation using malloc() and dealloc() functions. But, in case of C# developers no need to take care of GC and it's not recommended as well.    
+As said above, every programming language has their own way of implementing GC. In C programming, developers need to take care of memory allocation and deallocation using malloc() and dealloc() functions. But, in case of C# developers do not need to take address GC and it's also not a recommended practice.    
```

Comment:
on this line, it should be either `need to address GC` or it should be reverted back to the previous like `need to take care of GC`.
`need to take address GC` just seems odd and I feel it is grammatically wrong.